### ISSUE TITLE
Disk space saving 

### ISSUE THREAD

**Issue by shicks-seismo** [ThomasLecocq/SeismoRMS#23]
For disk space saving, could we have an option to only download the mseed if the corresponding .npz file isn't already existing? As far as I can see the .npz file doesn't actually depend on user-defined parameters so I don't think these ever really will change (unless the waveform data changes).
That way, the user could delete the .mseed files at the end of every run. This might be quite useful if regularly updating a station or running for different time periods.

**Comment by pjkoelemeijer** [comment]
Agreed - that would be nice! I keep deleting the mseed files and just download them again for disk space saving

**Comment by lmoresi** [comment]
See my comment on #21 - only downloading the mseed file if the npz needs regenerating. Nothing clever in what I suggest but a starting point for a real solution.

**Comment by megies** [comment]
Personally, I was only processing our own data and replaced the whole downloading with a SDS Client.. but probably thats not fitting most people, so just as a side comment.

**Comment by FMassin** [comment]
> Personally, I was only processing our own data and replaced the whole downloading with a SDS Client.. but probably thats not fitting most people, so just as a side comment.

Ha! I actually had to do this myself too: 
[paperzero.obspy.py.zip](https://github.com/ThomasLecocq/SeismoRMS/files/4657058/paperzero.obspy.py.zip)
